src/deserialization.hpp

```diff
@@ -210,6 +215,36 @@ class ConcreteTensorProtoDeserializator {
         }
     }
 
+    static ov::Tensor deserializeTensorProto(
+        const InferenceTensor& requestInput,
+        const std::shared_ptr<TensorInfo>& tensorInfo) {
+        OVMS_PROFILE_FUNCTION();
+        switch (tensorInfo->getPrecision()) {
+        case ovms::Precision::FP64:
+        case ovms::Precision::FP32:
+        case ovms::Precision::FP16:
+        case ovms::Precision::I64:
+        case ovms::Precision::I32:
+        case ovms::Precision::I16:
+        case ovms::Precision::I8:
+        case ovms::Precision::U64:
+        case ovms::Precision::U32:
+        case ovms::Precision::U16:
+        case ovms::Precision::BOOL:
+        case ovms::Precision::U1:
```

Comment:
Do we test U1 / BOOL precision? I have added those precisions to unsupported in serialization.